ckan/lib/redis.py

```diff
@@ -0,0 +1,63 @@
+# encoding: utf-8
+
+u'''
+Redis utilities.
+
+.. versionadded:: 2.6
+'''
+
+from __future__ import absolute_import
+
+import datetime
+import logging
+
+from pylons import config
+from redis import ConnectionPool, Redis
+
+
+log = logging.getLogger(__name__)
+
+REDIS_URL_SETTING_NAME = u'ckan.redis.url'
+
+REDIS_URL_DEFAULT_VALUE = u'redis://localhost:6379/0'
```

Comment:
If we're requiring redis now is it safe to assume we can connect to a redis running on localhost? How about we fail loudly if the value isn't set instead?